mlflow/utils/virtualenv.py

```diff
@@ -231,18 +232,59 @@ def _get_virtualenv_name(python_env, work_dir_path, env_id=None):
     )
 
 
-def _create_virtualenv(
-    local_model_path, python_bin_path, env_dir, python_env, extra_env=None, capture_output=False
-):
+def _get_virtualenv_activate_cmd(env_dir: Path):
     # Created a command to activate the environment
     paths = ("bin", "activate") if not is_windows() else ("Scripts", "activate.bat")
     activate_cmd = env_dir.joinpath(*paths)
-    activate_cmd = f"source {activate_cmd}" if not is_windows() else str(activate_cmd)
+    return f"source {activate_cmd}" if not is_windows() else str(activate_cmd)
+
 
+def _create_virtualenv(
+    local_model_path: Path,
+    python_env: _PythonEnv,
+    env_dir: Path,
+    pyenv_root_dir: Optional[str] = None,
+    env_manager: Literal["virtualenv", "uv"] = em.UV,
```

Comment:
```suggestion
    env_manager: Literal["virtualenv", "uv"] = em.VIRTUALENV,
```

can we make the default virtualenv because uv is still experimental?

Reply:
Ok, we can keep the default value, but suggest users to try uv. And update after it's stable